oraichain/assetlist.json

```diff
@@ -33,23 +33,496 @@
         },
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.png",
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.svg",
-          "theme": {
-            "circle": true,
-            "dark_mode": true,
-            "primary_color_hex": "#FFFFFF"
-          }
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.svg"
         },
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai.png",
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai.svg",
-          "theme": {
-            "circle": true,
-            "dark_mode": false,
-            "primary_color_hex": "#000000"
-          }
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai.svg"
         }
       ]
+    },
+    {
+      "description": "The ATOM native token of Oraichain.",
+      "denom_units": [
+        {
+          "denom": "ibc/A2E2EEC9057A4A1C2C0A6A4C78B0239118DF5F278830F50B4A6BDD7A66506B78",
+          "exponent": 0
+        },
+        {
+          "denom": "ATOM",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/A2E2EEC9057A4A1C2C0A6A4C78B0239118DF5F278830F50B4A6BDD7A66506B78",
+      "name": "ATOM",
+      "display": "atom",
+      "symbol": "ATOM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/atom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/atom.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/atom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/atom.svg"
+        }
+      ],
+      "coingecko_id": "cosmos"
```

Comment:
Please do not include coingecko_id on assets except for the original mintage (cosmoshub::uatom already has this defined)

Reply:
Please remove this for all ibc assets. Although it was removed for ATOM, I still see this on OSMO and others.